<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userInterface</class>
 <widget class="QWidget" name="userInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户</string>
  </property>
  <widget class="QLabel" name="label_balance">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>余额:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_info">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>390</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>个人信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_send_parcel">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>490</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发快递</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_inquiry">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>590</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>快递查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_welcome">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎使用</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_show_balance">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>861</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_main">
    <widget class="QTableWidget" name="tableWidget_show_parcel_recv">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>681</width>
       <height>281</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>待签收快递：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_recv">
     <property name="geometry">
      <rect>
       <x>731</x>
       <y>224</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>签收选中快递</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_inquiry_parcel">
    <widget class="QCheckBox" name="checkBox_time">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>341</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>时间</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_receiver">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>341</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>接收人</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sender">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>361</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>发送人</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_user_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>341</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_parcel_id">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>361</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>快递ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_time">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>341</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sender">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>361</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_courier">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>361</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>快递员</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_receiver">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>341</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_by_keyword">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>350</y>
       <width>82</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>条件查询</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_courier">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>360</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_user_name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_parcel_id">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_parcel_send">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>150</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>我发送的</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_parcel_recv">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>210</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>我接收的</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_parcel_all">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>100</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>全部快递</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_show_myparcel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>721</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="page_send_parcel">
    <widget class="QLabel" name="label_receiver">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>接收人：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_receiver_sendParcel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_description">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>快递描述：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_description_sendParcel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_select_parcel_type">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>130</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_parcel_type">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>快递种类：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_parcel_num">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>200</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>快递数量(kg/本)：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_parcel_num">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>210</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_title_send_parcel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送快递：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cost">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>预计花费金额：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ensure">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>360</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认发送</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cost">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注意：快递不足1kg部分按1kg算</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_person_info">
    <widget class="QListWidget" name="listWidget_person_info">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>256</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_balance_charge">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>余额充值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_charge_balance">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>充值金额：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_balance_to_charge">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_charge_balance">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>充值</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_change_pwd">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>修改密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_new_pwd">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>新密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_new_pwd">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>280</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cahnge_pwd">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>330</y>
       <width>82</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button_main">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>300</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>主页面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_log_out">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>30</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
